src/sentry/plugins/sentry_mail/models.py

```diff
@@ -171,11 +171,13 @@ def notify(self, notification):
             else:
                 for committer in committers:
                     for commit, score in committer['commits']:
-                        commit_data = commit.copy()
-                        commit_data['shortId'] = commit_data['id'][:7]
-                        commit_data['author'] = committer['author']
-                        commit_data['subject'] = commit_data['message'].split('\n', 1)[0]
-                        commits.append(commit_data)
+                        if commit['id'] not in commits:
```

Comment:
for some reason I though this code was actually executing in tests. It doesnt seem to be so I'm going to try to get in some basic coverage first.